Flagsmith-engine/Identity/Models/IdentityFeatures.cs

```diff
@@ -0,0 +1,10 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Flagsmith_engine.Identity.Models
+{
+    public class IdentityFeatures
```

Comment:
I'm not sure what this class is here for?

Reply:
Thanks to identify , this has to be removed.